spring-pulsar-spring-cloud-stream-binder/build.gradle

```diff
@@ -11,5 +11,9 @@ dependencies {
         exclude group: 'javax.annotation', module: 'javax.annotation-api'
     }
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testImplementation project(':spring-pulsar-test')
+    testImplementation 'org.testcontainers:pulsar'
```

Comment:
If we change these 2 deps to `api` in the `spring-test` lib then all users of the lib can remove these 2 deps. 

Don't worry about that in this PR though - just noticed it.